tests/manage/pv_services/test_cephfs_with_mds_network_failure.py

```diff
@@ -0,0 +1,94 @@
+import logging
+import time
+import pytest
+from random import randint
+from ocs_ci.ocs import constants
+from threading import Thread
+from ocs_ci.ocs.cluster import get_mds_standby_replay_info
+from ocs_ci.helpers.helpers import disable_vm_network_for_duration
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.framework.pytest_customization.marks import (
+    vsphere_platform_required,
+    tier4b,
+    skipif_external_mode,
+)
+from ocs_ci.ocs.resources.pod import search_pattern_in_pod_logs
+from ocs_ci.helpers.helpers import change_vm_network_state
+
+log = logging.getLogger(__name__)
+
+
+@skipif_external_mode
+@vsphere_platform_required
+class TestCephFSWithMDSNetworkFailure:
+    def teardown(self):
+        """
+        teardown function, Re-enabling the network connectivity of the VM if it was not done.
+        """
+        change_vm_network_state(self.standby_replay_node_ip, connect=True)
```

Comment:
Enable the vm network only if the network is down. lets try to avoid forcefully connecting the network though it is up.

Reply:
Before doing enable or disable vm network we actually checking the network state then we are changing the state.
Find the code in below link
https://github.com/red-hat-storage/ocs-ci/pull/7412/files#diff-69e40bb721a45bfa33dae338b44f5c90cdb0e68c6f19db835bccd5fe931015efR1591-R1595 

Let me know if we still want to add the condition.

Reply:
then we are good, Resolving this comment.